### PR TITLE
Feature/rename package

### PR DESCRIPTION
**Pull request by SethNguyenXBOT**

### CODE REVIEW

File: Competition2017/src/competition/operator_interface/OperatorInterface.java [hunk 1]
```diff
@@ -31,7 +31,8 @@ public OperatorInterface(WPIFactory factory, RobotAssertionManager assertionMana
         
         leftJoystick = factory.getJoystick(1);
         rightJoystick = factory.getJoystick(2);
-     
+        operatorJoystick = factory.getJoystick(3);
```
Comment (WasabiFan) [inline Competition2017/src/competition/operator_interface/OperatorInterface.java]: What's happening here?

File: Competition2017/src/competition/operator_interface/OperatorCommandMap.java [hunk 2]
```diff
@@ -97,7 +97,7 @@ public void setupShiftCommand(
            ShiftGearCommand shiftLow, 
            ShiftGearCommand shiftHigh)
    {
-       shiftLow.setGear(Gear.LOW_GEAR);
+       shiftLow.setGear(Gear.LOW_GEAR);
```
Comment (WasabiFan) [inline Competition2017/src/competition/operator_interface/OperatorCommandMap.java]: Remove trailing whitespace

File: Competition2017/src/competition/operator_interface/OperatorCommandMap.java [hunk 3]
```diff
@@ -97,7 +97,8 @@ public void setupShiftCommand(
            ShiftGearCommand shiftLow, 
            ShiftGearCommand shiftHigh)
    {
-       shiftLow.setGear(Gear.LOW_GEAR);
+   
+       shiftLow.setGear(Gear.LOW_GEAR);
```
Comment (WasabiFan) [inline Competition2017/src/competition/operator_interface/OperatorCommandMap.java]: What happened here?

File: Competition2017/src/competition/operator_interface/OperatorInterface.java [hunk 4]
```diff
@@ -31,7 +31,8 @@ public OperatorInterface(WPIFactory factory, RobotAssertionManager assertionMana
         
         leftJoystick = factory.getJoystick(1);
         rightJoystick = factory.getJoystick(2);
-     
+        operatorPanelJoystick = factory.getJoystick(3);
```
Comment (WasabiFan) [inline Competition2017/src/competition/operator_interface/OperatorInterface.java:34]: I suspect there's an existing joystick for the panel; this looks like a bad merge.

**Review by JohnGilb — APPROVED** [review]

File: Competition2017/src/competition/operator_interface/OperatorInterface.java (hunk 4 above)
Comment (JohnGilb) [reply Competition2017/src/competition/operator_interface/OperatorInterface.java:34]: Removed

**Pull request merged**